index.js

```diff
@@ -0,0 +1,86 @@
+const myLibrary = [];
```

Comment:
## Optional Changes
- [ ] Can we have a way of toggling the `read` and `unread` portion of our book. Right now users are confused as to what the  `false` bit stands for in the book display.
- [ ] Is it possible to add titles to the book fields getting displayed? We only have values, hard to tell what they stand for.
![library](https://user-images.githubusercontent.com/22345961/110998993-bdeeb880-8390-11eb-962e-d525a37784d3.png)


- [ ] Would you mind adding the functionality of `editing`, and `deleting` the book once created? That would be nice to have.